hosts

```diff
@@ -418,15 +418,8 @@
 
 # bazaarvoice.com
 0.0.0.0 analytics-static.ugc.bazaarvoice.com
-0.0.0.0 apps.bazaarvoice.com
-0.0.0.0 apps.nexus.bazaarvoice.com
 0.0.0.0 bv-analytics-js-prod.s3.amazonaws.com
 0.0.0.0 bv-analytics-js-qa.s3.amazonaws.com
-0.0.0.0 magpie-a.bazaarvoice.com
-0.0.0.0 magpie.bazaarvoice.com
-0.0.0.0 magpie-static.ugc.bazaarvoice.com
-0.0.0.0 magpie-stg-a.bazaarvoice.com
-0.0.0.0 magpie-stg.bazaarvoice.com
 0.0.0.0 network-a.bazaarvoice.com
 0.0.0.0 network.bazaarvoice.com
 0.0.0.0 network-eu-a.bazaarvoice.com
```

Comment:
Unless we can point to somewhere these are being used for tracking, I think we should ditch these too.